contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -164,6 +193,14 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
         }
 
         uint256 totalBurnedTxFees = BURNED_TX_FEES_ADDRESS.balance;
+
+        uint256 scaledAmount;
+        if (multiplyOnReceive) {
+            scaledAmount = totalBurnedTxFees / tokenMultiplier;
```

Comment:
Perhaps consider creating a helper function for calcuating scaledAmount since we do this more than once.